docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -0,0 +1,296 @@
+EOF
+```
+
+3. Confirm that the `hypershift-addon` is installed by running the following command:
+  
+```bash
+$ oc get managedclusteraddons -n <managed-cluster-used-as-hosting-service-cluster> hypershift-addon
+NAME               AVAILABLE   DEGRADED   PROGRESSING
+hypershift-addon   True
+```
+
+Your HyperShift add-on is installed and the management cluster is available to manage HyperShift hosted clusters.
+
+## Provision a HyperShift hosted cluster on AWS
+
+After installing the HyperShift operator and enabling an existing cluster as a hosting service cluster, you can provision a hypershift hosted cluster via the hypershift CLI.
+
+1. Set the following environment variables
+
+```bash
+export REGION=us-east-1
+export CLUSTER_NAME=clc-dhu-hs1
+export INFRA_ID=clc-dhu-hs1-01
+export BASE_DOMAIN=dev09.red-chesterfield.com
+export AWS_CREDS=$HOME/dhu-aws
+export PULL_SECRET=/Users/dhuynh/dhu-pull-secret.txt
+export BUCKET_NAME=acmqe-hypershift
+export BUCKET_REGION=us-east-1
+export INFRA_OUTPUT_FILE=$HOME/hostedcluster_infra.json
+export IAM_OUTPUT_FILE=$HOME/hostedcluster_iam.json
+```
+Note: in order for the cluster to show correctly in the MCE console UI, it is recommended to keep the `CLUSTER_NAME` and `INFRA_ID` the same.
+
+2. Ensure you are logged into your hub cluster.
+
+3. Create the infrastructure first:
+
+```bash
+hypershift create infra aws --name $CLUSTER_NAME \
+    --aws-creds $AWS_CREDS \
+    --base-domain $BASE_DOMAIN \
+    --infra-id $CLUSTER_NAME \
+    --region $REGION \
+    --output-file $INFRA_OUTPUT_FILE
+```
+
+4. In order to create the related IAM pieces, we'll need some info from the infra output.
+
+```bash
+cat test-infra.json                                                                                             
+{
+  "region": "us-east-1",
+  "zone": "",
+  "infraID": "clc-dhu-hs1",
+  "machineCIDR": "10.0.0.0/16",
+  "vpcID": "vpc-07541ba034d5a26a1",
+  "zones": [
+    {
+      "name": "us-east-1a",
+      "subnetID": "subnet-0e17ee93963c8bf34"
+    }
+  ],
+  "securityGroupID": "sg-0da668fbe75efeb43",
+  "Name": "clc-dhu-hs1",
+  "baseDomain": "dev09.red-chesterfield.com",
+  "publicZoneID": "Z00953301HRDK0M0YKQGE",
+  "privateZoneID": "Z02815081BZYZVASX5HII",
+  "localZoneID": "Z01446842N9Z8MWQL967F",
+  "proxyAddr": ""
+}
+```
+Specically, we'll need the `publicZoneID`, `privateZoneID`, and `localZoneID`
+
+5. Create the IAM:
+
+```bash
+hypershift create iam aws --infra-id $CLUSTER_NAME \
+    --aws-creds $AWS_CREDS \
+    --oidc-storage-provider-s3-bucket-name $BUCKET_NAME \
+    --oidc-storage-provider-s3-region $BUCKET_REGION \
+    --region $REGION \
+    --public-zone-id Z00953301HRDK0M0YKQGE \
+    --private-zone-id Z02815081BZYZVASX5HII \
+    --local-zone-id Z01446842N9Z8MWQL967F \
+    --output-file $IAM_OUTPUT_FILE
+```
+
+6. We're now ready to construct the necessary hosted cluster resources using the information created from the last 2 commands:
```

Comment:
Running `hypershift create cluster aws` does `create infra` and `create iam` internally so you don't need to do them separately. You run `create infra` and `create iam` separately so that you can just create `HostedCluster` and `NodePool` CR yourself without running the `hypershift create cluster aws` command.